### DISCUSSION TITLE
How to completely uninstall InfiniteRemote

### DISCUSSION THREAD

**Discussion by hungchieu** [infiniteremote/installer#7] [Q&A]
Hi,
Is there any script to completely remove InfiniteRemote?
Thank you.

**Comment by RickkeeC** [comment]
me too - i need to start over

**Comment by gitondezhuts** [comment]
I can work on an uninstaller, ideally use docker if testing on an existing server or reinstall the server, it shouldnt be used by anything else as the script can cause issues for existing installations like apache etc

**Comment by gitondezhuts** [comment]
https://github.com/infiniteremote/installer/wiki/Uninstall
